go.mod

```diff
@@ -10,13 +10,14 @@ require (
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
+	github.com/kubernetes-csi/external-snapshotter/client/v2 v2.0.0-00010101000000-000000000000
 	github.com/prometheus/client_golang v1.0.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.4.1
 	google.golang.org/grpc v1.28.0
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
+	k8s.io/api v0.18.5
+	k8s.io/apimachinery v0.18.5
```

Comment:
Any reason why api and apimachinery are bumped to v0.18.5?  Other k8s components are still at v0.18.0.

Reply:
When i run command "go mod tidy"  it automatically taking the latest version for api and apimachinery. Although i have tried to downgrade the version but go mod tidy automatically  upgrade the version for api and apimachinery

Reply:
Update to v0.19.0-rc.0